CONTRIBUTING.md

```diff
@@ -258,8 +258,9 @@ Additionally, there is an ability to disable certain components of the e2e suite
 
 To get started:
 
-- Run `make test-e2e`
-- Inspect the logs of the docker containers and see if something it’s there
+- Run `make test-e2e-debug` for the full e2e test suite.
+- Run `make test-e2e-short` for the short e2e test suite that excludes upgrade, IBC and state-sync tests.
```

Comment:
does `make test-e2e` run everything and clear containers?

Reply:
answers yes! We should keep that as recommended method in the readme right?

(PR is good to merge either way imo, just wanted to bring this up in case you wanted to change it)

Reply:
Yes, that's correct.

There are 2 differences between `make test-e2e` and `make test-e2e-debug`:
- The former attempts to clear everything after running 
- The former utilizes Go cache

Reply:
I think recommending `make test-e2e-debug` might be preferable because it is useful to keep the containers for debugging locally. I also find that it is helpful to avoid using Go cache for local debugging.

Both of them will clear/remove the containers before running the test too